week2/Program.cs

```diff
@@ -0,0 +1,29 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "This is dotnet week2 Assignment");
+
+app.MapGet("/wordFrequencyCount", () =>
+{
+    string inputString = "the quick Brown fox jumps over the lazy dog The";
+    inputString = inputString.ToLower();
+    var Words = inputString.Split(' ');  
+
+    Dictionary<string, int> DuplicateWordCount = new Dictionary<string, int>();
```

Comment:
the name of the variable should be with `camelCase` 😊

Reply:
Yah.. you are right. I will update.